calicoctl/calico_ctl/node.py

```diff
@@ -87,6 +87,9 @@
 CALICO_NETWORKING_ENV = "CALICO_NETWORKING"
```

Comment:
Hey @ishidawataru [I made a PR to your branch](https://github.com/ishidawataru/calico-containers/pull/1) to get the ENV/cmdline how I think it should be. I put a description in that PR. Take a look and merge into your branch if you agree.